package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "@boatbomber/highlighter",
+    "version": "1.0.0",
+    "license": "MIT",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/boatbomber/Highlighter.git"
+    },
+    "contributors": [
+        "BoatBomber"

```

Comment:
My name has no capitals, just `boatbomber` not `BoatBomber`. Other than that, looks fine to me!